packages/pwa-buildpack/package.json

```diff
@@ -7,7 +7,7 @@
   "description": "Build/Layout optimization tooling and Peregrine framework adapters for the Magento PWA",
   "main": "dist/index.js",
   "scripts": {
-    "build": "babel src --out-dir dist --root-mode 'root' --source-maps --copy-files",
+    "build": "cross-env babel src --out-dir dist --root-mode 'root' --source-maps --copy-files",
```

Comment:
@zetlen do we want to include `BABEL_ENV=production` here like in the peregrine commands?